oslo/meetup.yaml

```diff
@@ -0,0 +1,69 @@
+cfpLink: ""
+meetupID: Cloud-Native-and-Kubernetes-Oslo
+city: Oslo
+country: norway
+name: Cloud Native and Kubernetes Oslo
+photo: https://secure.meetupstatic.com/photos/event/9/3/1/3/highres_461497651.jpeg
```

Comment:
photo, city, country, name belongs to the autogenerated side now :)
I need to doc this better

the same goes for the IDs dates/addresses, etc down below.